apps/server/src/modules/user-login-migration/service/user-login-migration.service.ts

```diff
@@ -71,6 +71,11 @@ export class UserLoginMigrationService {
 
 		const savedMigration: UserLoginMigrationDO = await this.userLoginMigrationRepo.save(userLoginMigration);
 
+		// userLoginMigration throws an error when saved, if this codeblock runs before the userLoginMigrationRepo.save method.
```

Comment:
When it throws an error then you code should not be executed. You have to catch the error..